doc/source/security/encrypting-system-tablespace.rst

```diff
@@ -0,0 +1,61 @@
+.. _encrypting-system-tablespace:
+
+==================================================================
+Encrypting the System Tablespace
+==================================================================
+
+|Percona Server| supports system tablespace encryption. The InnoDB system tablespace is encrypted with the Master key encryption. 
```

Comment:
@robgolebiowski It uses master key even with the (experimental) keyring encryption settings?

Reply:
@dutow it deos not. I think we will have a separate paragraph on keyring encryption and encryption threads. So for now,  for the time that keyring is still experimental, I think this is enough. Maybe later we will mention here that in case of keyring encryption it will be encrypted with keyring key and link to keyring encryption paragraph. Or should we add it now ? @patrickbirch what do you think ?

Reply:
|Percona Server| supports system tablespace encryption. The InnoDB system
  tablespace may be encrypted with the Master key encryption or may use keyring
  encryption with :ref:`encrypting-threads`.